server/events/project_command_runner.go

```diff
@@ -185,6 +185,10 @@ func (p *DefaultProjectCommandRunner) runSteps(steps []valid.Step, ctx models.Pr
 			out, err = p.ApplyStepRunner.Run(ctx, step.ExtraArgs, absPath)
 		case "run":
 			out, err = p.RunStepRunner.Run(ctx, step.RunCommand, absPath)
+		case "var":
+			out, err = p.RunStepRunner.Run(ctx, step.RunCommand, absPath)
```

Comment:
Lets create a new `EnvStepRunner` that returns `envVarName, envVarValue, err`

Reply:
done